evap/staff/tools.py

```diff
@@ -194,6 +194,6 @@ def merge_users(main_user, other_user, preview=False):
 def find_next_unreviewed_course(semester, excluded):
     return semester.course_set.exclude(pk__in=excluded) \
         .exclude(state='published') \
-        .annotate(num_unreviewed_textanswers=Count("contributions__textanswer_set", filter=Q(contributions__textanswer_set__state=TextAnswer.NOT_REVIEWED), distinct=True)) \
-        .filter(num_unreviewed_textanswers__gt=0) \
+        .filter(contributions__textanswer_set__state=TextAnswer.NOT_REVIEWED) \
+        .annotate(num_unreviewed_textanswers=Count("contributions__textanswer_set")) \
```

Comment:
I have no idea whether the `distinct` is necessary here, as I don't know what it does exactly

Reply:
in my small example it was necessary, otherwise i got duplicated results. the reason i cannot explain exactly to you, i think it's because of the join that produces one line (and thus one object in the resulting querySet) per unreviewed textanswer, and not per course.

hm i see that you still need the annotate for the ordering, i thought that can be removed as well. well then the simpler filter is not that much of a simplification but oh well.